### ISSUE TITLE
Dungeon TBC - Black Morass

### ISSUE THREAD

**Issue by Saesuri** [Hazor-Hellscream/Project-Hellscream-Info#785]
After the 1st boss is defeated Chrono Lord Deja (17879), which is the 6/18 portal, the other portals take a really long time to spawn, about 2 minutes each. If you wait 2 minutes for every portal after 6/18, we're talking about a 25 minute dungeon in total, and thats unrealistic for a dungeon this size. This 2 minute spawn time for a portal should only happen after you defeat a boss, after the boss is defeated the spawn rate of the portals should go back to the normal rate.

Tl;dr: If a portal spawns a boss, next portal should spawn in 2 minutes. 
If a portal spawns a regular mob, next portal should spawn in a few seconds.

Aeonus doesn't seem to cast Thrash or Cleave (40504)
Mobs are lacking dialogs and weapons.
Npc at the entrance of the dungeon should give a quest, currently it does not.

Priority:
High. 

Sort of doable, but on live server people will not wait this amount of time to do this dungeon.
